package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
use this package.json
https://github.com/NourSammour/biletiniz/blob/master/package.json

and just change `name` to `biletiniz-api` and keep `scripts` section